syntax = "proto3";

message LocationMessage {
  int32 id = 1;
  int32 person_id = 2;
  float longitude = 3;
  float latitude = 4;
  string creation_time = 5;
}

message IdMessage {
  int32 id = 1;
}

service LocationService {
    rpc Create(LocationMessage) returns (LocationMessage);
    rpc Get(IdMessage) returns (LocationMessage);
}